client/packages/system/src/Name/DetailModal/DetailModal.tsx

```diff
@@ -68,16 +68,30 @@ export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
             <DetailInputWithLabelRow
               label={t('label.website')}
               inputProps={{ value: data?.website, disabled: isDisabled }}
+              inputAlignment="start"
               DisabledInput={
-                <>
-                  <MuiLink
-                    href={data.website ?? undefined}
-                    target="_blank"
-                    rel="noopener"
-                  >
-                    {data.website}
-                  </MuiLink>
-                </>
+                <MuiLink
+                  href={data.website ?? undefined}
+                  target="_blank"
+                  rel="noopener"
+                  sx={{
+                    // truncate long links
+                    overflow: 'hidden',
+                    textWrap: 'nowrap',
+                    textOverflow: 'ellipsis',
+                    // Make it look like another disabled text input consistence with other text
+                    // input components in this modal
+                    width: '198',
```

Comment:
Was there a reason for these odd numbers? 👀 Could we have them align with the other fields? 
![Screenshot 2024-05-08 at 14 09 06](https://github.com/msupply-foundation/open-msupply/assets/61820074/b6738a72-f2be-4711-9be1-d849d76d8b57)

Reply:
these numbers worked for me and were required if we would use a ellipsis to shorten the URL. But as said not happy about it. Another option would be to wrap the url. Is this what you show in the screenshot?

Reply:
Yup!